fastlane/Fastfile

```diff
@@ -307,49 +325,70 @@ platform :android do
-        release_status: 'draft',
-        skip_upload_apk: true,
-        skip_upload_metadata: true,
-        skip_upload_changelogs: true,
-        skip_upload_images: true,
-        skip_upload_screenshots: true,
-        json_key: UPLOAD_TO_PLAY_STORE_JSON_KEY
+        release_status: 'draft'
       )
       release_assets << aab_artifact_path
 
       signed_apk_artifact_path = signed_apk_artifact_path(app, version)
       download_universal_apk_from_google_play(
         package_name: APP_PACKAGE_NAME,
-        version_code: version_code_for_app(build_code, app),
+        version_code: version_code_for_app(app: app, version_code: build_code),
         destination: signed_apk_artifact_path,
         json_key: UPLOAD_TO_PLAY_STORE_JSON_KEY
       )
       release_assets << signed_apk_artifact_path
     end
 
-    create_gh_release(version: version, prerelease: is_beta, release_assets: release_assets.compact) if options[:create_gh_release]
+    create_gh_release(version: version, prerelease: is_beta, release_assets: release_assets.compact) if create_gh_release
+  end
+
+  # Update the rollout of all 3 variants (app, automotive, wear) of the current version to the given value
+  #
+  # @param [Float] percent the rollout percentage, between 0 and 1
+  #
+  lane :update_rollouts do |percent:|
+    version = version_name_current
+    build_code = build_code_current
+    is_beta = beta_version?(version)
+
+    not_found_variants = []
+    APPS.each do |app|
+      track = play_store_track(app: app, is_beta: is_beta)
+      version_code = version_code_for_app(app: app, version_code: build_code)
+      upload_to_play_store(
+        **UPLOAD_TO_PLAY_STORE_COMMON_OPTIONS,
+        skip_upload_aab: true,
+        track: track,
+        version_code: version_code,
+        release_status: percent.to_f < 1.0 ? 'inProgress' : 'completed',
+        rollout: percent.to_s
+      )
+    rescue FastlaneCore::Interface::FastlaneError => e
+      raise unless e.message =~ /Unable to find the requested release on track/
+
+      not_found_variants << app
+    end
+
+    not_found_variants.each do |app|
+      UI.important("'#{app}' variant for `#{version}` was not found in Google Play Console")
+    end
+    UI.user_error!('None of the 3 app variants were found in Google Play Console. We expected at least one') if not_found_variants.count == APPS.count
   end
 
-  lane :finalize_release do |options|
+  # @param skip_confirm [Boolean] If true, avoids any interactive prompt
+  #
+  lane :finalize_release do |skip_confirm: false|
     UI.user_error!('Please use `finalize_hotfix_release` lane for hotfixes') if release_is_hotfix?
     ensure_git_branch(branch: '^release/') # Match branch names that begin with `release/`
     ensure_git_status_clean
 
     UI.important("Finalizing release: #{release_version_current}")
-    unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+    unless skip_confirm || UI.confirm('Do you want to continue?')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/6a0928aa27197a8a7fec52a5f2f55ad056cabff3/fastlane/Fastfile#L391"></span>Style/IfUnlessModifier: Favor modifier <code>unless</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>